deployment/https/nginx.conf

```diff
@@ -41,7 +41,7 @@ http {
     listen [::]:80;
     listen 80;
 
-    server_name lj.starwit.de;
+    server_name pb.starwit.de;
```

Comment:
I think this config should not be in the official lirejarp package but added after the fact with a volume. Because this deployment is highly specific.
An example however would be usefull.

Reply:
new ticket ab#76